src/graphorm/index.ts

```diff
@@ -203,6 +204,14 @@ const obj = new GraphORM({
           parentColumn: 'userId',
         },
       },
+      total: {
+        select: (_, alias, qb) =>
+          qb
+            .select('COALESCE(COUNT(*), 0)')
+            .from(UserTopReader, 'utp')
+            .where(`utp."userId" = ${alias}."userId"`)
+            .groupBy('utp."userId"'),
+      },
```

Comment:
I went with the easiest solution here, and just do count directly of the table.
Alternatively my other plan was to extend the UserStats MV to include the count of total number of badges.

Let me know if I should go the MV route

Reply:
Going to use MV for this

Reply:
Refactored to MV